docs/consensus.md

```diff
@@ -1,9 +1,125 @@
+Github](https://github.com/nspcc-dev/neo-go/releases) or use [Docker
+image](https://hub.docker.com/r/nspccdev/neo-go). It has everything included,
+no additional plugins needed.
+
+Take appropriate (mainnet/testnet) configuration [from the
+repository](https://github.com/nspcc-dev/neo-go/tree/master/config) and save
+in some directory (we'll assume that it's available in the same directory as
+neo-go binary).
+
+### Configuration and execution
+
+Add the following subsection to `ApplicationConfiguration` section of your
+configuration file (`protocol.mainnet.yml` or `protocol.testnet.yml`):
+```
+  UnlockWallet:
+    Path: "wallet.json"
+    Password: "welcometotherealworld"
+```
+where `wallet.json` is a path to your NEP-6 wallet and `welcometotherealworld`
+is a password to your CN key. Run the node in a regular way after that:
+
+```
+$ neo-go node --mainnet --config-path ./
+```
+where `--mainnet` is your network (can be `--testnet` for testnet) and
+`--config-path` is a path to configuration file directory. If the node starts
+fine it'll be logging events like synchronized blocks. The node doesn't have
+any interactive CLI, it only outputs logs so you can wrap this command in a
+systemd service file to run automatically on system startup.
+
+Notice that the default configuration has RPC and Prometheus services enabled,
+you can turn them off for security purposes or restrict access to them with a
+firewall. Carefuly review all other configuration options to see if they meet
+your expectations. Details on various configuration options (as well as CLI
+commands) are provided [here](cli.md).
+
+### Registration
+
+To register as a candidate use neo-go as CLI command with an external RPC
+server for it to connect to (for chain data and transaction submission). You
+can use any public RPC server or an RPC server of your own like the node
+started at previous step. We'll assume that you're running the next command on
+the same node in default configuration with RPC interface available at port
+10332.
+
+Candidate registration is performed via NEO contract invocation that costs
+1000 GAS, so your account must have enough of it to pay. You need to provide
+your wallet and address to neo-go command:
+```
+$ neo-go wallet candidate register -a NiKEkwz6i9q6gqfCizztDoHQh9r9BtdCNa -w wallet.json -r http://localhost:10332
+```
+where `NiKEkwz6i9q6gqfCizztDoHQh9r9BtdCNa` is your address, `wallet.json` is a
+path to NEP-6 wallet file and `http://localhost:10332` is an RPC node to
+use.
+
+This command will create and send appropriate transaction to the network and
+you should then wait for it to settle in a block. If all goes well it'll end
+with "HALT" state and your registration will be completed.
+
+### Voting
+
+After registration completion if you own some NEO you can also vote for your
+candidate to help it become CN and receive additional voter GAS. To do that
+you need to know the public key of your candidate, if you the wallet it can be
```

Comment:
Is this a typo? `if you the wallet it...`

Reply:
It is. Fixed.